zulipterminal/cli/run.py

```diff
@@ -330,15 +331,19 @@ def main(options: Optional[List[str]]=None) -> None:
                 sys.exit(1)
             boolean_settings[setting] = (zterm[setting][0] == valid_values[0])
 
-        color_depth = int(args.color_depth)
+        if args.color_depth:
+            color_depth = int(args.color_depth)
+        else:
+            color_depth = int(zterm['color-depth'][0])
```

Comment:
We should likely do some validation here - this crashes if it is not one of the valid values.

Reply:
WIll do that..

Reply:
And should the program exit if it is not one of those values or just tell the user that "this is not one of the valid values" and start zulip terminal with the default of 256?

Reply:
It's a little unclear, but my suggestion would be that since we validate color depth on the command line (via argparse choices) and reject bad values before running, we should likely do the same before running via a config-file option. If a user has specified a value in the config, then it's likely they have (tried to) do it intentionally, and running anyhow may give the impression the config file value is valid and correct. Feedback much like from argparse with wrong values, indicating good values, should be reasonable.

Reply:
So the program should not quit if a bad value is given in the variable `color-depth`?

Reply:
If there is a bad value from the command-line, argparse gives a message (give it a try!)

If there is a bad value from the config-file, we should give a similar message.